tests/test_hsm.py

```diff
@@ -727,7 +727,7 @@ def _doComputeImage(self, position=None, color=None):
             # inconsistent.  Old shapeHSM code attempted to infer the former
             # from the latter, but was unreliable.  New code infers the former
             # directly, so this inconsistency no longer breaks things.
-            bbox.shift(geom.Extent2I(1, 1))
+            bbox.shift(geom.Extent2I(1, 2))
```

Comment:
Here, I've adjusted the shift from `Extent2I(1, 1)` to `Extent2I(1, 2)` to reduce confusion during debugging. This modification aids in distinguishing that some errors are not just related to the 0 vs 1 indexing issue.

Reply:
This is good. It's always better to have different values in `x` and `y` to catch any errors that may coincidentally cancel each other.